cmd/virt-v2v-monitor/virt-v2v-monitor.go

```diff
@@ -65,6 +157,9 @@ func main() {
 	// Start prometheus metrics HTTP handler
 	klog.Info("Setting up prometheus endpoint :2112/metrics")
 	http.Handle("/metrics", promhttp.Handler())
+	klog.Info("Setting up prometheus endpoint :2112/vm_config")
```

Comment:
i think it's not a prometheus endpoint

Reply:
yeah I was thinking if created a new endpoint for that or using the existing one even though its not in the category